functions/consumer/xmpp-consumer/src/test/docker-compose/xmpp/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.7"
+services:
+  Openfire:
+    image: fishbowler/openfire:latest
+    ports:
+      - "5222:5222"
```

Comment:
This is the standard port it uses for clients to connect. So internally, it will need to be mapped to that port, ultimately.  How do you propose to proceed?

Reply:
Take a look at the source code for `org.testcontainers.containers.MongoDBContainer` and you will see how they use a mapped port. Another option would be to create an xmpp TC impl like `MongoDBContainer` rather than the docker compose route. 

As for proceeding - this does not have to be done in the scope of this PR as long as we have an issue created to follow up and do that in some reasonable subsequent time period. :) What you have works and we should get this in and iterate on it from there. My 2c.